app/src/main/res/layout/character_detail_layout.xml

```diff
@@ -1,76 +1,106 @@
 
     <TextView
+        android:id="@+id/more_details_title"
         style="@style/Letters"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/see_the_comics_label" />
+        android:text="@string/more_details_label"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/character_description" />
 
     <TextView
-        android:id="@+id/character_description"
+        android:id="@+id/character_details_link"
         style="@style/Body"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/more_details_title" />
 
     <TextView
+        android:id="@+id/character_appearances_title"
         style="@style/Letters"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/more_details_label" />
-
-    <TextView
-        android:id="@+id/character_name"
-        style="@style/Title"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content" />
+        android:text="@string/appearances_label"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/character_details_link" />
 
     <TextView
-        android:id="@+id/character_comics_link"
+        android:id="@+id/character_appearances_comics"
         style="@style/Body"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        app:layout_constraintLeft_toLeftOf="@id/character_details_link"
+        app:layout_constraintTop_toBottomOf="@id/character_appearances_title" />
 
     <TextView
+        android:id="@+id/character_comics"
         style="@style/Letters"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/description" />
+        android:text="@string/see_the_comics_label"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/character_appearances_comics" />
 
     <TextView
-        style="@style/Letters"
+        android:id="@+id/character_comics_link"
+        style="@style/Body"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/appearances_label" />
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/character_comics" />
+
+    <ProgressBar
+        android:id="@+id/loading_bar"
+        style="@style/FragmentLoadingBarStyle"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
the constraints are not necessary here because you are using match_parent. please remove them

Reply:
Done thx